server/tests/transactions.go

```diff
@@ -66,7 +66,7 @@ func (t Transaction) NewRun() TransactionRun {
 	return TransactionRun{
 		TransactionID:      t.ID,
 		TransactionVersion: t.GetVersion(),
-		CreatedAt:          time.Now(),
+		CreatedAt:          time.Now().UTC(),
```

Comment:
This was a bug, we need to always use UTC from the server

Reply:
why? I don't think we set any TZ on any other place